lib/tbd/psi.rb

```diff
@@ -1487,7 +1487,7 @@ def process(model = nil, argh = {})
 
         edge[:surfaces].keys.each do |i|
           break     if is[:rimjoist] || is[:balcony]
-          break unless deratables.size == 2
+          break unless deratables.size > 0
```

Comment:
Revised logic (e.g. for spaces with multipliers), where floors are e.g. adiabatic. In such cases, there's only one _deratable_ surface per edge (i.e. the exterior wall above, sharing the same edge with the floor). The fix ensures that "rimjoist" thermal bridge types are assigned to (e.g. adiabatic) floor edges (not "transition"). An oversight that should have been caught/tested in the previous release.